sound/soc/sof/intel/cnl.c

```diff
@@ -166,7 +166,7 @@ irqreturn_t cnl_ipc_irq_thread(int irq, void *context)
 
 		/* handle messages from DSP */
 		if ((hipctdr & SOF_IPC_PANIC_MAGIC_MASK) == SOF_IPC_PANIC_MAGIC) {
-			struct sof_intel_hda_dev *hda = sdev->pdata->hw_pdata;
+			struct sof_intel_hda_dev *hdev = sdev->pdata->hw_pdata;
```

Comment:
I think it's super-confusing @ujfalusi 

traditionally hdev is used for this
````c
struct hdac_device *hdev
````

so that's why we used 'hda' to avoid confusions.

@ranj063 can you chime in on this?

Reply:
right, so then use `hda` for `struct sof_intel_hda_dev` ?

Reply:
I think so but you want a second opinion from @ranj063 !